src/pages/SetupPlot.vue

```diff
@@ -252,19 +253,22 @@ export default defineComponent({
       const dirExists = await native.dirExists(this.plotDirectory)
 
       if (dirExists) {
+        console.log("SETUP PLOT | found the old plotting directory")
         const files = await tauri.fs
           .readDir(this.plotDirectory)
           .catch(console.error)
 
         if (files) {
           console.log("FILES ARE: :", files)
-          if (files.length === 0 || !files.some(item => item.name === "chains" )) {
+          if (files.length === 0 ||  files.length === 1 && files.some(item => item.name === "subspace-desktop.cfg")){
```

Comment:
Please never ever use `a || b && c`, I don't have to remember what the priority of operators should be in this case in a specific language.